### PR TITLE
Added instructor outline for functions

### PR DESCRIPTION
**Pull request by yaseminreis**

### CODE REVIEW

File: lessons/13_bootcamp_functions/functions_instructor_outline.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+## What is a function?
```
Comment (pab2163) [inline lessons/13_bootcamp_functions/functions_instructor_outline.md:1]: Thanks, I think this looks great! @ashiq-r31 probably good to start with some sort of visual diagram of functions (maybe using some examples like `print()` or `len()` that students already know) at the beginning of class?

File: lessons/13_bootcamp_functions/functions_instructor_outline.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+ * We can break out of the function or return a value with `return`
+ * `print` vs `return` - fundamentally different in their applications!
+    
+```python
+num = multiply(2, 3)
+num = multiply(multiply(2, 3), 4)
+print(num)
+```
+
+ ## 4. Built-in functions
```
Comment (pab2163) [inline lessons/13_bootcamp_functions/functions_instructor_outline.md:56]: I wonder whether it makes sense to talk about these sooner, before we show how to make our own functions? No strong opinions though

Comment (ashiq-r31) [reply lessons/13_bootcamp_functions/functions_instructor_outline.md:56]: Yeah re-ordering this section to the top might be better

File: lessons/13_bootcamp_functions/functions_instructor_outline.md (hunk 1 above)
Comment (ashiq-r31) [reply lessons/13_bootcamp_functions/functions_instructor_outline.md:1]: something like `input -> function -> output`

**Comment by ashiq-r31** [comment]
I'm going to merge this, will make some updates before class though

**Pull request merged**